tests/rules/javascript/test.content_scripts_file_absent.js

```diff
@@ -15,10 +15,8 @@ function createJsScanner(code, validatedFilename, existingFiles = {}) {
 describe('content_scripts_file_absent', () => {
   it('should show an error when content script is missing', async () => {
```

Comment:
Nit, it could be reasonable to mention in the test description that the content script missing path has to be "absolute" (e.g. something like "should show an error for a missing absolute content script path")

Reply:
Fixed in 473b1a6